### DISCUSSION TITLE
How can I add description to the root of Generated Document?

### DISCUSSION THREAD

**Discussion by huyhoang160593** [RobinTail/express-zod-api#1318] [Q&A] [answered]
I have tried to find the description to pass when create new Docuement object but can't find it anywhere. The document doesn't have it too: https://www.npmjs.com/package/express-zod-api#creating-a-documentation

**Comment by RobinTail** [comment, marked as answer]
The `Documentation` class extents the `OpenApiBuilder` (3.0) from `openapi3-ts` library.
According to the type of the `OpenApiObject`, top level description that you refer to, is most likely the one located in `info` object:
https://github.com/metadevpro/openapi3-ts/blob/e470c7c7c60b7c75e24e1c824ca49e55d025d9ef/src/model/openapi30.ts#L24

You can set it and any other properties you need to the instance of `Documentation`. Try this, @huyhoang160593 :

```ts
const documentation = new Documentation({ ... });
documentation.info.description = "Your description here";
console.log( documentation. getSpecAsYaml() );
```
